llvm/test/CodeGen/AArch64/basic-pic-auth.ll

```diff
@@ -0,0 +1,82 @@
+; RUN: llc -mtriple=aarch64-linux-gnu -global-isel=0 -fast-isel=0         -verify-machineinstrs \
```

Comment:
A filename that starts with `ptrauth-` might be better to group ptrauth tests together.
I am not sure the test should focus specifically on PIC. It's common to test both non-pic and pic in one test.